### DISCUSSION TITLE
General SSL security question.

### DISCUSSION THREAD

**Discussion by sirganya** [uNetworking/uWebSockets.js#226] [Q&A] [answered]
Assuming I have a client and a server with both IP's mapped directly to real network interfaces and I establish a secure connection between them, once that connection is made, are there any security vulnerabilities I should watch out for during the life of the connection? Assuming certs are secure.

**Comment by ghost** [comment, marked as answer]
You can run tests such as https://www.ssllabs.com/ssltest/ on your server. Other than that no, but OpenSSL is not exactly without flaws
